docs/root/configuration/security/_include/sds_source_example.yaml

```diff
@@ -0,0 +1,98 @@
+static_resources:
+
+  clusters:
+    - name: sds_server_mtls
+      typed_extension_protocol_options:
+        envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+          "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+          explicit_http_config:
+            http2_protocol_options:
+              connection_keepalive:
+                interval: 30s
+                timeout: 5s
+      load_assignment:
+        cluster_name: sds_server_mtls
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: 127.0.0.1
+                  port_value: 8234
+      transport_socket:
+        name: envoy.transport_sockets.tls
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+          common_tls_context:
+            tls_certificates:
+            - certificate_chain:
+                filename: certs/sds_cert.pem
+              private_key:
+                filename: certs/sds_key.pem
+    - name: sds_server_uds
+      typed_extension_protocol_options:
+        envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+          "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+          explicit_http_config:
+            http2_protocol_options: {}
+      load_assignment:
+        cluster_name: sds_server_uds
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                pipe:
+                  path: /tmp/uds_path
+    - name: example_cluster
+      connect_timeout: 0.25s
+      load_assignment:
+        cluster_name: local_service_tls
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: 127.0.0.1
+                  port_value: 8443
+      transport_socket:
+        name: envoy.transport_sockets.tls
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+          common_tls_context:
+            tls_certificate_sds_secret_configs:
+            - name: client_cert
+              sds_config:
+                resource_api_version: V3
+                api_config_source:
+                  api_type: GRPC
+                  transport_api_version: V3
+                  grpc_services:
+                  - google_grpc:
+                      target_uri: unix:/tmp/uds_path
```

Comment:
not sure if this will matter exactly, but the path `/tmp/uds_path` wont exist

the configs are tested 2 ways - firstly just with static analysis, and secondly it will try to start a server with the config (altho doesnt make any requests or test any further)

not sure if this will pass the server test which happens later in ci

Reply:
I verified configs locally to the point that envoy was able to start without functionality, we can probably check this with CI